packages/core-sdk/src/seaport/interface.ts

```diff
@@ -0,0 +1,100 @@
+import { abis } from "@bosonprotocol/common";
+import { Interface } from "@ethersproject/abi";
+import { BigNumberish } from "@ethersproject/bignumber";
+
+const seaportIface = new Interface(abis.SeaportABI);
+
+export enum eOrderType {
+  FULL_OPEN = 0,
+  PARTIAL_OPEN = 1,
+  FULL_RESTRICTED = 2,
+  PARTIAL_RESTRICTED = 3,
+  CONTRACT = 4
+}
+
+export enum eItemType {
+  NATIVE = 0,
```

Comment:
let's try to not use the value 0 if possible and if it makes sense

Reply:
These values are the ones used in the smart contract. So no way to change them